core/API/DataTableManipulator/Flattener.php

```diff
@@ -196,6 +196,11 @@ private function flattenRow(
                 $subDimension = $report->getThirdLeveltableDimension();
```

Comment:
Should we refactor to use the getNthLevelDimension here as well? If not, why not?

Reply:
@michalkleiner No we shouldn't refactor here, as there will be reports which would be using this function and not the `getNthLevelDimension` which is the new function.